### ISSUE TITLE
Swagger UI crashes with RangeError: Maximum call stack size exceeded` for nested 10K-item response

### ISSUE THREAD

**Issue by miro-mlynarik** [tiangolo/fastapi#3914]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
import uvicorn

app = FastAPI()

@app.get('/api/data', tags=['Data'])
def map_data():
    mapping = {}
    for i in range(17000):
        mapping[i] = [
            'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 0.9988,
            'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 0.9988,
            'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 0.9988
        ]
    return mapping


if __name__ == '__main__':
    uvicorn.run("sqlreader:app", host="127.0.0.1", port=5000)
```

### Description

* Open the browser and call the endpoint /data
* The web UI starts to get the JSON, but after a several tens of seconds, it finishes with error `RangeError: Maximum call stack size exceeded` 

### Operating System

Linux

### FastAPI Version

0.65.2

### Python Version

3.8.5

**Comment by Mause** [comment]
This sounds like an issue with swagger ui, not fastapi?

**Comment by miro-mlynarik** [comment]
> This sounds like an issue with swagger ui, not fastapi?

Yes, I was considering also reporting it under Swagger repository, but also wanted to get some insight here first, because when I run the same endpoint using [Connexion](https://github.com/zalando/connexion) framework, it runs without problems. So I suppose there might be somehow problem with specific combination of FastAPI and Swagger. 

![image](https://user-images.githubusercontent.com/44208384/133881658-81cc72b9-3d3a-43db-82c4-eecdd7a8d811.png)
